packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -150,7 +147,7 @@ describe('ui:create:atomic', () => {
       args.shift()!,
       args,
       {
-        cwd: normalizedProjectDir,
+        cwd: originalProjectDir,
```

Comment:
We start the project from the "og" directory because rollup is confused by nested junctions (which only occurs here)